metadata-transformer/src/test/java/no/unit/nva/metadata/CreatePublicationRequestTest.java

```diff
@@ -38,6 +46,14 @@ private CreatePublicationRequest sampleRequest() {
         return request;
```

Comment:
doesNotHaveEmptyValues() was unstable. Not caused by my changed. Fixed it by ensuring refrence was set